resources/backend/js/refresh-resources.js

```diff
@@ -109,24 +109,70 @@ function convertKitRefreshResources( button ) {
 				}
 			);
 
-			// Populate select options from response data.
-			response.data.forEach(
-				function ( item ) {
-					let label = '';
-					switch ( resource ) {
-						case 'forms':
-							label = item.name + ' [' + ( item.format !== '' ? item.format : 'inline' ) + ']';
-								break;
-						default:
-							label = item.name;
-						break;
-					}
-
-					// Add option.
-					const option = new Option( label, item.id, false, selectedOption == item.id );
-					document.querySelector( field ).add( option );
-				}
-			);
+			// Depending on the resource we're refreshing, populate the <select> options.
+			switch ( resource ) {
+				case 'restrict_content':
+					// Populate select `optgroup`` from response data, which comprises of Tags and Products.
+					// Tags.
+					response.data.tags.forEach(
+						function ( item ) {
+							document.querySelector( field + ' optgroup[data-resource=tags]' ).appendChild(
+								new Option(
+									item.name,
+									'tag_' + item.id,
+									false,
+									( selectedOption == 'tag_' + item.id ? true : false )
+								)
+							);
+						}
+					);
+
+					// Products.
+					response.data.products.forEach(
+						function ( item ) {
+							document.querySelector( field + ' optgroup[data-resource=products]' ).appendChild(
+								new Option(
+									item.name,
+									'product_' + item.id,
+									false,
+									( selectedOption == 'product_' + item.id ? true : false )
```

Comment:
The ternary operator here (and on lines 154 and 168) is unnecessary. Also wondering if there's any reason not to use the triple equals.